pkg/sanity/node.go

```diff
@@ -29,6 +29,26 @@ import (
 	. "github.com/onsi/gomega"
 )
 
+func isNodeCapabilitySupported(c csi.NodeClient,
+	capType csi.NodeServiceCapability_RPC_Type,
+) bool {
+
+	caps, err := c.NodeGetCapabilities(
+		context.Background(),
+		&csi.NodeGetCapabilitiesRequest{})
+	Expect(err).NotTo(HaveOccurred())
+	Expect(caps).NotTo(BeNil())
+	Expect(caps.GetCapabilities()).NotTo(BeNil())
+
+	for _, cap := range caps.GetCapabilities() {
+		Expect(cap.GetRpc()).NotTo(BeNil())
```

Comment:
Expect(cap.GetRpc()).NotTo(BeNil()) is used just for a brevity, right? Same could be done just in code..

Reply:
yep, just following the style of the rest of the file.